docs/configuration-parameters.md

```diff
@@ -210,6 +210,22 @@ RESET
+| carbon.number.of.cores.while.loading      | Specifies number of cores to be used while loading data. **NOTE:** Refer to [Data Loading Configuration](#data-loading-configuration)#carbon.number.of.cores.while.loading for detailed information. |
+| carbon.number.of.cores.while.compacting   | Specifies number of cores to be used while compacting data. **NOTE:** Refer to [Data Loading Configuration](#data-loading-configuration)#carbon.number.of.cores.while.compacting for detailed information. |
+| enable.offheap.sort                       | To enable off-heap memory usage. **NOTE:** Refer to [Data Loading Configuration](#data-loading-configuration)#enable.offheap.sort for detailed information. |
+| carbon.blockletgroup.size.in.mb           | Specifies the size of each blocklet group. **NOTE:** Refer to [Data Loading Configuration](#data-loading-configuration)#carbon.blockletgroup.size.in.mb for detailed information. |
+| carbon.enable.auto.load.merge             | To enable compaction along with data loading. **NOTE:** Refer to [Compaction Configuration](#compaction-configuration)#carbon.enable.auto.load.merge for detailed information. |
+| carbon.major.compaction.size              | To configure major compaction size. **NOTE:** Refer to [Compaction Configuration](#compaction-configuration)#carbon.major.compaction.size for detailed information. |
+| carbon.compaction.level.threshold         | To configure compaction threshold. **NOTE:** Refer to [Compaction Configuration](#compaction-configuration)#carbon.compaction.level.threshold for detailed information. |
+| carbon.enable.vector.reader               | To enable fetching data as columnar batch of size 4*1024 rows instead of fetching a row at a time. **NOTE:** Refer to [Query Configuration](#query-configuration)#carbon.enable.vector.reader for detailed information. |
+| enable.unsafe.in.query.processing         | To enable use of unsafe functions while scanning the data during query. **NOTE:** Refer to [Query Configuration](#query-configuration)#enable.unsafe.in.query.processing for detailed information. |
+| carbon.push.rowfilters.for.vector         | To enable complete row filters handling by carbon in case of vector. **NOTE:** Refer to [Query Configuration](#query-configuration)#carbon.push.rowfilters.for.vector for detailed information. |
+| carbon.query.stage.input.enable           | To make query to include staged input files. **NOTE:** Refer to [Query Configuration](#query-configuration)#carbon.query.stage.input.enable for detailed information. |
+| carbon.enable.tablestatus.backup          | To enable table status file backup. **NOTE:** Refer to [System Configuration](#system-configuration)#carbon.enable.tablestatus.backup for detailed information. |
```

Comment:
Have you tried setting this as checking ?
Just because it is dynmicConfigurable in CCC, doesn't mean it is supported. 
There has to be an entry in SessionParams.validateKeyValue()

I tried this in cluster

0: jdbc:hive2://localhost:10000> set carbon.enable.tablestatus.backup=true;
Error: org.apache.carbondata.core.exception.InvalidConfigurationException: The key carbon.enable.tablestatus.backup not supported for dynamic configuration. (state=,code=0)
0: jdbc:hive2://localhost:10000>

Reply:
if we can add a test case, it will be better

Reply:
Modified and added a testcase now

Reply:
Modified and added a testcase